runtime/kusama/src/lib.rs

```diff
@@ -1034,12 +1034,6 @@ construct_runtime! {
 	}
 }
 
-impl pallet_babe::migrations::BabePalletPrefix for Runtime {
```

Comment:
These should have been removed in https://github.com/paritytech/polkadot/pull/2764.